src/components/system/SystemAlert.jsx

```diff
@@ -76,6 +77,18 @@ class Alert extends Component {
       }];
     }
 
+    if (system.serviceWorkerUpdated) {
+      alertMessages.push({
+        message: `Thers is a newer version of this app`,
```

Comment:
typo, should best "There".  Also, let's say, "There is a newer version of APZU IC3 app, please close and reopen he app to update."

Reply:
@jomadoye actually, I forget, this is not APZU IC3 specific :) so "There is a newer version of this app, please close and reopen the app to update."